src/commands/server/deploy.ts

```diff
@@ -427,3 +456,14 @@ export default class Deploy extends Command {
     }
   }
 }
+
+export async function askForChectlUpdateIfNeeded(): Promise<void> {
```

Comment:
Maybe  Should this method be located in the VersionHelper or some util class? What do you think?

Reply:
Yes, I had thought on that before submitting the PR. I'll move the method to utils.